pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -567,6 +608,9 @@ func (c *Cluster) provisionPodTemplateSpec(osdProps osdProperties, restart v1.Re
 // Currently we can't mount a block mode pv directly to a privileged container
 // So we mount it to a non privileged init container and then copy it to a common directory mounted inside init container
 // and the privileged provision container.
+
+// DON'T USE SYMLINK BUT DIRECTLY MOUNT THE BLOCK IN THE CEPH PATH?
```

Comment:
yes, hoping for this change!

Reply:
I need to do a more experiment and this requires a change in c-v

Reply:
No need! Here's the commit that does it: 016a494c19bf6028c03232224dd48dde6986c3db, I made it a separate commit in case we decide not to do it.